v2/pkg/commands/build/base.go

```diff
@@ -513,17 +513,27 @@ func (b *BaseBuilder) BuildFrontend(outputLogger *clilogger.CLILogger) error {
 	frontendDir := filepath.Join(b.projectData.Path, "frontend")
 
 	// Check there is an 'InstallCommand' provided in wails.json
-	if b.projectData.InstallCommand == "" {
+	var installCommand string
```

Comment:
I think this can be simplified with the helper method you created! 🚀

Reply:
🚀 Thanks, I've simplified the logic of determining the installation command.